src/main/java/io/vertx/core/http/StreamPriority.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2011-2018 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.http;
+
+import io.netty.handler.codec.http2.Http2CodecUtil;
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * This class represents HTTP/2 stream priority defined in RFC 7540 clause 5.3
+ */
+@DataObject
+public class StreamPriority {
+    
+    public static final StreamPriority DEFAULT = new StreamPriority();
```

Comment:
not needed

Reply:
This is still needed for default value in the VertxHttp2Stream.